### PR TITLE
Vsc creation

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Tests: 
No possibility to create a hvdc vsc from an existing one : no search button on top right of the dialog ...

File: src/components/dialogs/network-modifications/vsc/creation/vsc-creation-dialog.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
+import React, { useCallback, useEffect, useState } from 'react';
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/network-modifications/vsc/creation/vsc-creation-dialog.js:1]: add copyright

File: src/components/results/dynamicsimulation/dynamic-simulation-result-chart.js [hunk 2]
```diff
@@ -23,86 +23,75 @@ import TooltipIconButton from './common/tooltip-icon-button';
...
-const useStyles = makeStyles((theme) => ({
+const styles = {
     root: {
         display: 'flex',
         flexDirection: 'column',
         width: '100%',
         height: '100%',
     },
-    modal: {
+    modal: (theme) => ({
```
Comment (Walid-Sahnoun) [inline src/components/results/dynamicsimulation/dynamic-simulation-result-chart.js:43]: make (theme) at the top 
styles = (theme) for refactor

File: src/services/study/network-modifications.js [hunk 3]
```diff
@@ -1501,3 +1501,64 @@ export function updateSwitchState(studyUuid, currentNodeUuid, switchId, open) {
...
+        '/network-modifications';
+
+    if (isUpdate) {
+        createVscUrl += '/' + encodeURIComponent(modificationUuid);
+        console.info('Updating vsc creation');
+    } else {
+        console.info('Creating vsc creation');
+    }
+
+    console.log('dcNominalVoltage : ', dcNominalVoltage);
```
Comment (Walid-Sahnoun) [inline src/services/study/network-modifications.js:1536]: remove console.log

**Review by Walid-Sahnoun — CHANGES_REQUESTED** [review]
the three fields "DC nominal voltage", "DC resistance" and "Maximum active power"
they are optional, but in the US mentioned as required.
the field "Converters mode" the first option is selected by default, is this wanted ?
when we add new line in the reactive limit table is start by p1 but in the US is mentioned to start with p0.
when create VSC with empty DCNominalVoltage, an error in logs
VscCreationInfos.getDcNominalVoltage()" is null

**Review by FranckLecuyer — APPROVED** [review]

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
test ok

**Pull request merged**